src/classes/RD2_OpportunityEvaluation_BATCH.cls

```diff
@@ -58,6 +58,34 @@ public without sharing class RD2_OpportunityEvaluation_BATCH
         private set;
     }
 
+    /***
+    * @description Contains batch Apex class name
+    */
+    private String batchClassName {
+        get {
+            if (batchClassName == null) {
+                batchClassName = RD2_OpportunityEvaluation_BATCH.class.getName();
+            }
+            return batchClassName;
+        }
+        set;
+    }
+
+    /***
+    * @description Checks if another concurrent batch job is in progress
+    */
+    @TestVisible
+    private Boolean anotherBatchIsInProgress {
```

Comment:
The Boolean vars should start like a question, ie `is` and `has` and to be consistent with the `RD` batch var, let's rename it to `isBatchInProgress`.